tests/core/test_utilities.py

```diff
@@ -901,23 +901,49 @@ def test_has_module():
     assert not has_module('not_a_module')
 
 
-def test_fit_plane_to_points():
-    points = ex.load_airplane().points
-    plane, center, normal = fit_plane_to_points(points, return_meta=True)
-
-    assert np.allclose(normal, [-2.5999512e-08, 0.121780515, -0.99255705])
-    assert np.allclose(center, [896.9954860028446, 686.6470205328502, 78.13187948615939])
```

Comment:
Note that the in the new tests, the returned center does NOT have this value. This means that there was previously a bug with how center was calculated. This is now fixed by this PR (and verified  as correct by the tests)